app/views/admin/versions/index.html.haml

```diff
@@ -0,0 +1,48 @@
+.row
+  .col-md-12
+    .page-header
+      %h1 Revision History
+      %p.text-muted
+        Log of changes made to conferences and associated resources
+
+- @versions.each do |version|
+  .row
+    .col-md-9
+      %p
+        = version.whodunnit.nil? ? 'Someone (probably via the console)' : User.find(version.whodunnit).name
+
```

Comment:
Name is not a mandatory field. We should show something when there is no user.name